skipped_tests.toml

```diff
@@ -111,16 +125,19 @@ unit = [
 	"tests/unit/utils/test_templates.py::RenderTestCase::test_render_mako_variable",
 ]
 integration = [
+	"tests/integration/loader/test_ext_modules.py",  # Make pytest to stuck in sle15sp1 classic pkg
 	"tests/integration/modules/test_cmdmod.py::CMDModuleTest::test_which_bin",
+	"tests/integration/ssh/test_state.py::SSHStateTest::test_state_running",  # This test makes no sense with new locking mechanism for Salt SSH
```

Comment:
Can we drop the test if it does not make sense?

Reply:
Yeah, makes more sense.